src/Swashbuckle.AspNetCore.SwaggerGen/SchemaGenerator/MemberInfoExtensions.cs

```diff
@@ -39,7 +39,10 @@ public static bool IsNonNullableReferenceType(this MemberInfo memberInfo)
                 ? ((FieldInfo)memberInfo).FieldType
                 : ((PropertyInfo)memberInfo).PropertyType;
 
-            if (memberType.IsValueType) return false;
+            if (memberType.IsValueType)
+            {
+                return false;
+            }
```

Comment:
Could you please revert this change as it's stylistic not functional - I prefer the previous style

Reply:
Code change reverted